helpers/mathjax.js

```diff
@@ -138,15 +142,17 @@ export function loadMathJax(mathJaxConfig) {
 					const InputJax = startup.getInputJax();
 					const OutputJax = startup.getOutputJax();
 					const html = mathjax.document(root, { InputJax, OutputJax });
-					html.render();
+					html.render().typeset();
```

Comment:
When startup typesetting is disabled, we need to explicitly invoke the typeset action on the document.